File: Practice Codes/CC2 Java to Python/Finals/Exam/twitter.py
print("------------------------")
print("| Twitter Login System |")
print("------------------------")
print()

usernames = ["user1", "user2", "user3", "admin"]
passwords = ["pass1", "pass2", "pass3", "admin"]
full_names = ["name 1", "name 2", "name 3", "admin admin"]

tries = 1
found = False
while True:
    username = input("Enter username: ")
    password = input("Enter password: ")

    for i in range(len(usernames)):
        if username == usernames[i] and password == passwords[i]:
            found = True
            print(f">>> Login successful. Welcome back, {usernames[i].lower()}")
            print("-------------------")
            print("| Account Profile |")
            print("-------------------")
            print(f"Full name: {full_names[i].upper()}")
            print(f"Username: {usernames[i].lower()}")
            break

    if not found:
        if tries == 3:
            print("Too many attempts. You have been locked")
            break
        else:
            print("Invalid username or password. Try again")
            tries += 1

File: CC2_ComProg1/CC2_Finals/Lab Exam/Twitter.java
import java.util.Scanner;
/**
 * @author pao
 * BSCS 1B
 */
public class Twitter {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("------------------------");
        System.out.println("| Twitter Login System |");
        System.out.println("------------------------");
        System.out.println();
        String usernames[] = {"user1","user2","user3","admin"};
        String passwords[] = {"pass1","pass2","pass3","admin"};
        String fullnames[] = {"name 1","name 2","name 3","admin admin"};
        int tries = 1;
        boolean condi = true; 
        boolean isFound = false;
        while (condi) {
            System.out.print("Enter username : ");
            String inputUser = sc.nextLine();
            System.out.print("Enter password : ");
            String inputPass = sc.nextLine();
            for(int i = 0; i < usernames.length; i++) {
                if (inputUser.equals(usernames[i]) && inputPass.equals(passwords[i])) {
                    isFound = true;
                    System.out.println(">>> Login successful, Welcome back, " + usernames[i].toLowerCase());
                    System.out.println("-------------------");
                    System.out.println("| Account Profile |");
                    System.out.println("-------------------");
                    System.out.println("Fullname: " + fullnames[i].toUpperCase());
                    System.out.println("Username: " + usernames[i].toLowerCase());
                    condi = false;
                } 
            }
            if (!isFound) {
                if (tries == 3) {
                    System.out.println("Too many attempts. You have been locked");
                    condi = false;
                } else {
                    System.out.println("Invalid username or password. Try again");
                    tries++;
                }
            }   
        }
    }
}
